service/test/controllers/usermanagement/UserControllerTest.java

```diff
@@ -161,29 +115,21 @@ public void testcreateUserWithInvalidPhone() {
   }
 
   @Test
-  public void testupdateUserProfile() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
+  public void testUpdateUserProfileSuccess() {
     Map<String, Object> requestMap = new HashMap<>();
     Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.PHONE, "8800088000");
-    innerMap.put(JsonKey.COUNTRY_CODE, "+91");
-    innerMap.put(JsonKey.EMAIL, "abbc@gmail.com");
-    // innerMap.put(JsonKey.USERNAME , "userName");
-    innerMap.put(JsonKey.FIRST_NAME, "john");
-    innerMap.put(JsonKey.LAST_NAME, "rambo");
+    innerMap.put(JsonKey.USER_ID, userId);
+    innerMap.put(JsonKey.EMAIL, emailId);
+    innerMap.put(JsonKey.FIRST_NAME, firstName);
+    innerMap.put(JsonKey.LAST_NAME, lastName);
     List<String> roles = new ArrayList<>();
-    roles.add("user");
+    roles.add(role);
```

Comment:
In create or update user we are not accepting roles , so remove it from unit test as well.